exonum-java-binding/core/src/main/java/com/exonum/binding/core/blockchain/serialization/BlockSerializer.java

```diff
@@ -94,7 +94,7 @@ private static HashCode toHashCode(Hash hash) {
 
   @VisibleForTesting
   static ImmutableMap<String, ByteString> toHeadersMap(AdditionalHeaders headers) {
-    return headers.getHeaders().getEntryList()
+    return headers.getHeaders().getEntriesList()
```

Comment:
The most mysterious change in the entire PR

Reply:
I think the message was updated earlier (before BlockchainData merge, but after its implementation), and as no CI was configured — we just didn't notice during the merge.